README.md

```diff
@@ -574,3 +574,45 @@ print(new_name)  # ['Ryan', 'McDermott']
 
 **[⬆ back to top](#table-of-contents)**
 
+### Don't write to global functions
+
+Polluting globals is a bad practice in many languages because you could clash 
```

Comment:
I don't understand this one. The function `config()` is not global, but defined at the module level, which is fine to do.

However, you could do the following:
```
globals()['foo'] = 1
```
This would modify the global namespace, which is seldom necessary and would be considered bad practice (use modules instead).

Feel free to open a new pull request if I have misunderstood this section.

Reply:
I think you're right. Subsection adapted from https://github.com/jupeter/clean-code-php#dont-write-to-global-functions. I suppose PHP requires that you [explicitly declare namespaces](https://www.php.net/manual/en/language.namespaces.fallback.php), so the same module-scoping policy doesn't apply?